### ISSUE TITLE
[Bug] Configuration overrides are not honored

### ISSUE THREAD

**Issue by maciej-zuk** [jonlabelle/SublimeJsPrettier#295]
### Explain the issue

If you have prettierrc config with file type specific overrides, these overrides are not honored when prettier is run from SublimeJsPrettier. 

Caused by 10be0843c674a39778a7442e39975bb5cd5011f9. Not happening in 1.66.0

Please hide wrap in quotes behind config flag.

### Prettier version

3.2.5

### JsPrettier plug-in version

1.67.1

### Platform

Linux

### SublimeText version

4

### Prettier command

```console
/home/user/.nvm/versions/node/v20.10.0/bin/node /tmp/project/node_modules/.bin/prettier --config /tmp/project/.prettierrc --parser scss --use-tabs true --ignore-path /tmp/project/.prettierignore --stdin-filepath "/tmp/project/main.scss" --log-level debug
```

### Prettier config

```json
{
    "singleQuote": true,
    "overrides": [
        {
            "files": "*.scss",
            "options": {
                "singleQuote": false
            }
        }
    ]
}
```

### Steps to reproduce

1. create config with file specific overrides
2. run plugin to format such file
3. observe overrides not applied

### Reproducible outside of SublimeText?

No

### Attach a minimal project

[project.zip](https://github.com/jonlabelle/SublimeJsPrettier/files/15236297/project.zip)

**Comment by jonlabelle** [comment]
Hi @maciej-zuk. Yea, that's very weird, definitely an issue though. And enough for me to rollback #290 until further analysis.

## Output from w/ and w/out double-quotes

### With --stdin-filepath in double quotes

```console
-----------------------------------------
 JsPrettier DEBUG - Prettier CLI Command 
-----------------------------------------

/Users/jon/Downloads/project/node_modules/.bin/prettier --config /Users/jon/Downloads/project/.prettierrc --parser scss --use-tabs true --ignore-path /Users/jon/Downloads/project/.prettierignore --stdin-filepath "/Users/jon/Downloads/project/main.scss" --log-level debug

Prettier produced the following output:

[debug] normalized argv: {"":[],"cache":false,"color":true,"editorconfig":true,"useTabs":true,"config":"/Users/jon/Downloads/project/.prettierrc","parser":"scss","ignorePath":["/Users/jon/Downloads/project/.prettierignore"],"filepath":"\"/Users/jon/Downloads/project/main.scss\"","logLevel":"debug","configPrecedence":"cli-override","debugRepeat":0,"plugins":[],"_":[],"__raw":{"_":[],"cache":false,"color":true,"editorconfig":true,"use-tabs":true,"config":"/Users/jon/Downloads/project/.prettierrc","parser":"scss","ignore-path":"/Users/jon/Downloads/project/.prettierignore","stdin-filepath":"\"/Users/jon/Downloads/project/main.scss\"","log-level":"debug","config-precedence":"cli-override","debug-repeat":0,"plugin":[]}}
[debug] load config file from '/Users/jon/Downloads/project/.prettierrc'
[debug] loaded options `{"singleQuote":true}`
[debug] applied config-precedence (cli-override): {"filepath":"\"/Users/jon/Downloads/project/main.scss\"","singleQuote":true,"parser":"scss","useTabs":true}

Prettier process finished with exit code 0.
```

### Without --stdin-filepath in double quotes

```console
-----------------------------------------
 JsPrettier DEBUG - Prettier CLI Command 
-----------------------------------------

/Users/jon/Downloads/project/node_modules/.bin/prettier --config /Users/jon/Downloads/project/.prettierrc --parser scss --use-tabs true --ignore-path /Users/jon/Downloads/project/.prettierignore --stdin-filepath /Users/jon/Downloads/project/main.scss --log-level debug

Prettier produced the following output:

[debug] normalized argv: {"":[],"cache":false,"color":true,"editorconfig":true,"useTabs":true,"config":"/Users/jon/Downloads/project/.prettierrc","parser":"scss","ignorePath":["/Users/jon/Downloads/project/.prettierignore"],"filepath":"/Users/jon/Downloads/project/main.scss","logLevel":"debug","configPrecedence":"cli-override","debugRepeat":0,"plugins":[],"_":[],"__raw":{"_":[],"cache":false,"color":true,"editorconfig":true,"use-tabs":true,"config":"/Users/jon/Downloads/project/.prettierrc","parser":"scss","ignore-path":"/Users/jon/Downloads/project/.prettierignore","stdin-filepath":"/Users/jon/Downloads/project/main.scss","log-level":"debug","config-precedence":"cli-override","debug-repeat":0,"plugin":[]}}
[debug] load config file from '/Users/jon/Downloads/project/.prettierrc'
[debug] loaded options `{"singleQuote":false}`
[debug] applied config-precedence (cli-override): {"filepath":"/Users/jon/Downloads/project/main.scss","singleQuote":false,"parser":"scss","useTabs":true}

Prettier process finished with exit code 0.
```

**Comment by jonlabelle** [comment]
PR #290 has been [rolled back](https://github.com/jonlabelle/SublimeJsPrettier/commit/f9be844b51fc8c3c8b7f28616ccf2019126c660c).

Closes #295
